src/client/components/Editor.tsx

```diff
@@ -32,6 +34,37 @@ class Editor extends React.PureComponent<EditorProps> {
         }
     }
 
+  componentDidMount(): void {
+      // TODO: why does using gutterclick yield weird highlighting?
+      this.aceComponent.current.editor.on("gutterclick", function(e) {
+        var target = e.domEvent.target;
+
+        if (target.className.indexOf("ace_gutter-cell") == -1){
+            return;
+        }
+
+        if (!e.editor.isFocused()){
+            return;
+        }
+
+        if (e.clientX > 25 + target.getBoundingClientRect().left){
+            return;
+        }
+
+        var breakpoints = e.editor.session.getBreakpoints(row, 0);
+        var row = e.getDocumentPosition().row;
+
+        // If there's a breakpoint already defined, it should be removed, offering the toggle feature
+        if(typeof breakpoints[row] === typeof undefined){
```

Comment:
Just put `if (breakpoints[row] === undefined) {` (I know this is from the tutorial... That's such a weird way of writing that check in JS)